docs/developer.md

```diff
@@ -2,21 +2,19 @@
 
 The intended audience for this document is future developers of TWLight. Hi!
 
-To get set up with TWLight locally you can [twlight_vagrant](https://github.com/WikipediaLibrary/twlight_vagrant).
+To get set up with TWLight locally you will need [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/).
 
 ## Shell scripts
 
 A suite of shell scripts for performing various functions can be found in the /bin folder.
 
-Scripts starting with `virtualenv_` should be run as `www` (e.g. `sudo su www virtualenv_migrate.sh`).
+These scripts can be run with, for example `docker exec twlight virtualenv_migrate.sh`.
 
 ### Virtual environment
 
 To activate the Python virtual environment for the project, run:
 
-`sudo su www`
-
-`source /var/www/html/TWLight/bin/virtualenv_activate.sh`
+`/app/bin/virtualenv_activate.sh`
```

Comment:
Is this a thing that still needs to happen? `docker exec twlight bash` should already bring you into an activated environment.

Reply:
Ah, you're right. Removed that section.